### PR TITLE
Add FontAwesome local resources

### LINKED ISSUE #13717: Remove font-awesome 6.4 CDN from resources
### Describe the bug
font-awesome 6.4 does not have a CDN but one is defined in the resources. Instead, maybe use the local resource using `.SetUrl()`

https://www.jsdelivr.com/package/npm/@fortawesome/fontawesome-free

When I used font 6.4 in production, I get issue with the integrity token 
@agriffard

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

**Comment by hishamco** [comment]
Closed in favor of #13720

**Comment by MikeAlhayek** [comment]
@agriffard @hishamco I think line 379 should still be added even if we fixes the SRI

File: src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs [hunk 1]
```diff
@@ -376,10 +376,11 @@ ResourceManifest BuildManifest()
 
             manifest
                 .DefineStyle("font-awesome")
+                .SetUrl("~/OrchardCore.Resources/Vendor/fontawesome-free/css/all.min.css", "~/OrchardCore.Resources/Vendor/fontawesome-free/css/all.css")
                 .SetCdn("https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.0/css/all.min.css", "https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.0/css/all.css")
                 .SetCdnIntegrity("sha384-iw3OoTErCYJJB9mCa8LNS2hbsQ7M3C0EpIsO/H5+EGAkPGc6rk+V8i04oW/K5xq0", "sha384-kql1DN2uGoPEQ5t+GDSzWfg5WkqG8NmdGZSPRN9MTpb39cMBi40Ck3AROUEb+RqE")
                 .SetVersion("6.4.0");
-
+
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Resources/ResourceManagementOptionsConfiguration.cs:383]: Remove extra whitespace

**Review by hishamco — CHANGES_REQUESTED** [review]

**Comment by hishamco** [comment]
> @agriffard @hishamco I think line 379 should still be added even if we fixes the SRI

Why, unless you need an offline version, right?

**Comment by hishamco** [comment]
> Add FontAwesome local resource

While you change the title, I knew what you mean :) seems we asked for this from awhile, but for unknown reason the issue has been closed or there's a pending PR :)

**Comment by MikeAlhayek** [comment]
> > @agriffard @hishamco I think line 379 should still be added even if we fixes the SRI
> 
> Why, unless you need an offline version, right?

I don't think it's useless. It's used as a fallback if the CDN failed I believe.

**Comment by hishamco** [comment]
For me it's fine, but let us ensure that all the related resources needed by font-awesome are there, furthermore, we need to check it doesn't break the old versions

**Comment by hishamco** [comment]
> I don't think it's useless. It's used as a fallback if the CDN failed I believe.

Totally agree, seems my comment added after milliseconds of yours :)

**Review by agriffard — APPROVED** [review]

**Pull request merged**